.coveragerc

```diff
@@ -0,0 +1,28 @@
+[run]
```

Comment:
what is this file?

Reply:
Not sure, this is something for coverage not suave itself